inc/admin.php

```diff
@@ -90,9 +90,8 @@ function nkdeletepending_options_page() {
 			}
 		} ?>
 
-		<h2><?php _e( 'Delete Pending Comments', 'delete-pending-comments' ) ?></h2> <?php
-		require_once( 'nkuttler.php' );
```

Comment:
Delete this file as part of this commit.

Reply:
Fixed.